x-pack/plugins/alerting/README.md

```diff
@@ -414,6 +460,111 @@ features.registerKibanaFeature({
+
+For more information and other examples of subfeature privilege, refer to the [user documentation](https://www.elastic.co/guide/en/kibana/master/development-security.html#example-3-discover).
+
+```typescript
+features.registerKibanaFeature({
+  id: 'my-application-id',
+  name: 'My Application',
+  app: [],
+  alerting: [
+    'my-application-id.my-rule-type',
+    'my-application-id.my-other-rule-type'
+  ],
+  privileges: {
+    all: {
+      app: ['my-application-id', 'kibana'],
+      savedObject: {
+        all: [],
+        read: [],
+      },
+      alerting: {
+        rule: {
+          all: [
+            'my-application-id.my-rule-type',
+            'my-application-id.my-other-rule-type'
+          ]
+        },
+        alert: {
+          read: [
+            'my-application-id.my-rule-type',
+            'my-application-id.my-other-rule-type'
+          ]
+        }
+      },
+      ui: [],
+      api: [],
+    },
+    read: {
+      app: ['lens', 'kibana'],
+      alerting: {
+        rule: {
+          read: [
+            'my-application-id.my-rule-type',
+            'my-application-id.my-other-rule-type'
+          ]
+        },
+        alert: {
+          read: [
+            'my-application-id.my-rule-type',
+            'my-application-id.my-other-rule-type'
+          ]
+        }
+      },
+      savedObject: {
+        all: [],
+        read: [],
+      },
+      ui: [],
+      api: [],
+    },
+  },
+  subFeatures: [
+    {
+      name: 'Manage Alerts',
+      privilegeGroups: [
+        {
+          groupType: 'independent',
+          privileges: [
+            {
+              id: 'alert_manage',
+              name: 'Manage Alerts',
+              includeIn: 'all',
+              alerting: {
+                alert: {
+                  all: [
+                    'my-application-id.my-rule-type',
+                    'my-application-id.my-other-rule-type'
+                  ],
+                },
+              },
+              savedObject: {
+                all: [],
+                read: [],
+              },
+              ui: [],
+            }
+          ]
+        }
+      ]
+    }
+  ]
+});
+```
+
+In the above example, note that the base feature privilege grants users with the `all` role `all` access the rules of the specified rule types but only `read` access to the alerts of the same rule type. In order to get `all` access to the alerts of these rule types, the role must grant the `manage_alerts` subfeature privilege. Because the `manage_alerts` subfeature privilege has `includeIn` set to `all`, it is _automatically_ included in the `all` feature privilege but can be excluded when the role is defined.
```

Comment:
Should `manage_alerts` match the privilege id? So be `alert_manage` in this example?

Reply:
Good catch! Updated in this commit: https://github.com/elastic/kibana/pull/100127/commits/357c79189e7f10e00588a9934cf55ca03885a1f3